utilities/backup/backup_engine.cc

```diff
@@ -2079,14 +2077,9 @@ IOStatus BackupEngineImpl::CopyOrCreateFile(
     io_s = dest_writer->Append(data);
 
     if (rate_limiter != nullptr) {
-      if (!src.empty()) {
-        rate_limiter->Request(data.size(), Env::IO_LOW, nullptr /* stats */,
-                              RateLimiter::OpType::kWrite);
-      } else {
```

Comment:
Good catch 👍- not migrating OpType::kWrite in backup engine makes it easy to overlook this needed change.